doc/31-Permissions.md

```diff
@@ -0,0 +1,49 @@
+# Permissions
+
+Each permission in Icinga Web is denoted by a namespaced key, which is used to group permissions.
+The wildcard `*` can be used to grant all permissions in a certain namespace. Examples:
+
+* `training/assets`
+* `training/assets/users`
+* `training/*`
+
+In order to create a permission for an Icinga Web module, we register each permission in its `configuration.php` like so:
+
+```php
+<?php
+
+$this->providePermission(
+    'training/assets',
+    $this->translate('Allow to view and configure assets')
+);
+
+$this->providePermission(
+    'training/assets/users',
+    $this->translate('Allow to view and configure users of assets')
+);
+```
+
+We can now use these permissions in our Controllers by using the `assertPermission()` method. For example in the `init()` method which is called by the constructor. This would require the given permission for the entire Controller.
```

Comment:
```suggestion
We can now use these permissions in our Controllers by using the `assertPermission()` method. For example in the `init()` method which is called by the constructor. This would require a user to have the given permission for the entire Controller.
```